logins-sql/ffi/src/lib.rs

```diff
@@ -112,12 +112,13 @@ pub unsafe extern "C" fn sync15_passwords_delete(
     state: *const PasswordEngine,
     id: *const c_char,
     error: *mut ExternError
-) -> bool {
+) -> u8 {
```

Comment:
Hmm, we really need to share more of this code across projects :)

Reply:
Did you mean to comment on a different line?

Reply:
Oh, yeah, that's why I think the minimally safe low-level API should be exposed by us, and a higher level API should consume that and be exposed by a-c